docs/pages/develop/development-builds/introduction.mdx

```diff
@@ -1,88 +1,78 @@
+### Remote push notifications
 
-You can think of a development build as your fully customizable version of Expo Go, tailored specifically for your project. Or, conversely, Expo Go is a development environment with a preset runtime.
+While in-app [notifications](https://docs.expo.dev/versions/latest/sdk/notifications/) are available in Expo Go, remote push notifications (that is, sending a push notification from a server into the app) are not. This is because a push notification service should be tied to your own push notifications certificates and while it is possible to make it work in Expo Go, it often caused confusion for production builds. It is therefore better to test remote push notifications in Development Builds so you can ensure parity in behavior between development and production.
 
-</Collapsible>
+### Universal Links / App Links
 
-<Collapsible summary="What other types of builds are there?">
+Both [iOS Universal Links](https://docs.expo.dev/linking/ios-universal-links/) and [Android App Links](https://docs.expo.dev/linking/android-app-links/) require a two-way association between the native app and the website. In particular, it requires for the native app ro include the URL for the lined website. This is not possible with Expo Go due to the aforementioned native code immutability.
 
-- A [production build](/deploy/build-project/) is for the general public, deployed through stores.
-- A [preview build](/build/internal-distribution/) lets your team test your next release, either by installing an APK directly on Android or using ad hoc or enterprise provisioning on iOS.
+### (iOS Device only) open projects using older SDKs
 
-</Collapsible>
+Expo Go can only support one SDK version at a time. So, when we release a new SDK version we need to update Expo Go to support the new SDK version and this will be the only version of Expo Go available to install from the stores.
+
+If you're developing on Android Device, Android Emulator or iOS Simulator, a compatible version of Expo Go can be [downloaded and installed](https://expo.dev/go). The only platform where this is not possible is iPhone devices because apple does not support side-loading older versions of apps.
+
+## How to convert from Expo Go to a Development build
+
+To migrate from Expo Go, you'll need to
+
+1. install `expo-dev-client`
+2. build the native app using native build tools
+
+The details on how to do this based on the platform you're building for is covered in the [next section](/develop/development-builds/create-a-build).
+
+The Expo Dev Client package includes the launcher UI (shown in the screenshots below), dev menu, extensions to test updates etc. The Expo Go app has the dev menu built in, so we'll need to install it ourselves for a development build.
```

Comment:
```suggestion
The Expo Dev Client library includes the launcher UI (shown in the screenshots below), dev menu, extensions to test updates, and more. The Expo Go app has the dev menu built in, so we'll need to install it ourselves for a development build.
```

It's unclear as to what "so we'll need to install it ourselves for a development build." the last sentence means.

Reply:
Hm I'm not sure how to phrase it better. Expo Go had the dev tools built in. With dev builds we're no longer using Expo Go so we need to install the dev tools separately. Maybe changing it to "separately" instead of "ourselves" helps?

Reply:
ah okay, I wasn't aware of that. Yeah, changing the phrase might be more informative, like: "so you'll need to install it separately for a development build.". I'll leave this as a separate suggestion on the changes you've already implemented.